.gitignore

```diff
@@ -1,45 +1,48 @@
-# For PCBs designed using KiCad: http://www.kicad-pcb.org/
-# Format documentation: http://kicad-pcb.org/help/file-formats/
-
-*/media/*
-
-# Temporary files
-*.000
-*.bak
-*.bck
-*.kicad_pcb-bak
-*.sch-bak
-*~
-_autosave-*
-*.tmp
-*-save.pro
-*-save.kicad_pcb
-fp-info-cache
-
-# Netlist files (exported from Eeschema)
-*.net
-
-# Autorouter files (exported from Pcbnew)
-*.dsn
-*.ses
-
-# Exported BOM files
-*.xml
-*.PNG
-
-# CircuitPython drive files
-*.fseventsd/
-*.metadata_never_index
-*.Trashes
-*boot_out.txt
-*.URL
-sha.py
-*secrets*
-
-*.xlsx
-
-# pyc files
+# SPDX-FileCopyrightText: 2022 Kattni Rembor, written for Adafruit Industries
```

Comment:
we should add our own copyright here
2023 Thomas Damiani, written for Carnegie Mellon University's RExLab
maybe add a credit to the original author
With credit to Kattni Rembor of Adafruit Industries

something like that